### DISCUSSION TITLE
Termux Configuration Options!

### DISCUSSION THREAD

**Discussion by Vanillma** [termux/termux-app#3557] [Q&A] [answered]
Hi, I have been checking all the commands in the termux.configs file for a few days.
Unfortunately, I do not understand one of the commands!!
```
### Uncomment to let keyboard toggle button to enable or disable software
### keyboard instead of showing/hiding it.
# soft-keyboard-toggle-behaviour = enable/disable
```
(I googled and read the history of the Termux repository issues and the discussion about it.)
But I still could not understand what exactly this command does!?
Opening and closing the keyboard works fine on my Termux and no need to double click to open the keyboard!
(As far as I can think, this command seems to exist to fix this problem).
Can you explain a little more clearly?

**Comment by agnostic-apollo** [comment, marked as answer]
e4e638bd

**Reply by Vanillma** [reply]
Hey bro
This is about keyboard toggle button on the hard keyboard...right!?...I think I get it!!
And this option closes the soft keyboard And prevents it from opening!...When a keyboard toggle button is pressed on the hard keyboard!?

**Reply by agnostic-apollo** [reply]
Not hard keyboard, the `KEYBOARD` extra keys and button in left drawer.

**Reply by Vanillma** [reply]
Thank you... I tested it again and it worked as you said... Now I understand it deeply... :D
